docs/tutorials/qsimcirq_gcp.md

```diff
@@ -0,0 +1,357 @@
+
+```bash
+gcloud init
+```
+Respond to the prompts with your project ID and the zone where you created your VM. When completed, view the glcoud configuration:
+
+```bash
+gcloud config list
+```
+Output should look something like: 
+
+```less
+region = us-east4
+zone = us-east4-c
+[core]
+account = wellhello@gargle.com
+disable_usage_reporting = False
+project = quantum-22222
+```
+
+## Connect via port forward to your VM
+You are now ready co connect to your VM via a port forward gcloud command. Detailed instructions can be found [here](https://research.google.com/colaboratory/local-runtimes.html), but the following instructions should be sufficient to succeed. Connect up with SSH with port 8888 forwarded to your local machine:
+
+```bash
+gcloud compute ssh [YOUR_INSTANCE_NAME] -- -L 8888:localhost:8888
+```
+You should now see the command line prompt from your VM:
+```bash
+wellhello@qsim-1 ~ $ 
+```
+### Run the Jupyter / qsim container on your VM 
+At the command prompt you can now start a Docker container with all the required code to run simulations. Start the container:
+
+```bash
+docker run -v `pwd`:/homedir -p 8888:8888 gcr.io/quantum-builds/github.com/quantumlib/jupyter_qsim:latest
+```
+You should see several lines of output ending with lines like:
+```less
+    
+    To access the notebook, open this file in a browser:
+        file:///root/.local/share/jupyter/runtime/nbserver-1-open.html
+    Or copy and paste one of these URLs:
+        http://e1f7a7cca9fa:8888/?token=aa16e1b6d3f51c58928037d34cc6854dac47347dd4c0eae5
+     or http://127.0.0.1:8888/?token=aa16e1b6d3f51c58928037d34cc6854dac47347dd4c0eae5
+
+```
+Copy the last URL  in the output. Edit the URL to replace `127.0.0.1` with `localhost`. Save this URL for the next step. This URL points to your local runtime, running as a Docker container on your VM.
+
+## Connect Colab to your local runtime
+
+First, get a notebook up and running on Colab.
+
+ <a target="_blank" href="https://colab.research.google.com/github/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run a qsim notebook.</a>
+
+    
+  If you are connect to Colab with a notebook visible, you can click on `Connect` to get the UI:
+
+<img src="../images/colab_local.png" width="400"/>
+
+Select `Connect to local runtime`. You will see the UI:
+
+<img src="../images/colab_settings.png" width="500"/>
+
+This is where you past the edited URL from the previous section. Then click `Connect`
+
+<img src="../images/colab_success.png" width="300"/>
+
+You should now be able to run the cells in the notebook.
+
+### Large circuit
+A large circuit is provided with 32 qubits and a depth of 14 gate operations.  Run it here:
+
+
+<a target="_blank" href="https://colab.research.google.com/github/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run q32d14.ipynb.</a>
```

Comment:
This points to the qsimcirq tutorial notebook, not `q32d14.ipynb`. (It's OK if this link is broken until the PR is submitted - I was able to bring up the correct notebook by editing the URL to point to this branch)

Reply:
Fixed.